src/resources/sfml_loaders.h

```diff
@@ -37,25 +38,38 @@ el::ResourceLoader<ResourceType> makeResourceLoader(Func func,
   return el::ResourceLoader<ResourceType>(loader, key);
 }
 
-}  // namespace detail
+} // namespace detail
 
 template <typename ResourceType>
 // ResourceType: The type of resource we load.
-el::ResourceLoader<ResourceType> fromFile(const std::string& filename) {
-  return detail::makeResourceLoader<ResourceType>([=](ResourceType& resource) {
+el::ResourceLoader<ResourceType>
+fromFile(const nu::FilePath::StringType &filename) {
+  return detail::makeResourceLoader<ResourceType>([=](ResourceType &resource) {
+#if OS(POSIX)
     return resource.loadFromFile(filename);
+#else
+    std::string s(filename.begin(), filename.end());
+    return resource.loadFromFile(s);
+#endif // OS(POSIX)
   }, filename);
 }
 
 template <>
-el::ResourceLoader<sf::Texture> fromFile(const std::string& filename) {
-  return detail::makeResourceLoader<sf::Texture>([=](sf::Texture& texture) {
+el::ResourceLoader<sf::Texture>
+fromFile(const nu::FilePath::StringType &filename) {
+  return detail::makeResourceLoader<sf::Texture>([=](sf::Texture &texture) {
+#if OS(POSIX)
     bool success = texture.loadFromFile(filename);
+#else
+    std::string s(filename.begin(), filename.end());
+    bool success = texture.loadFromFile(s);
+#endif // OS(POSIX)
+
     texture.setSmooth(true);
     return success;
   }, filename);
 }
 
-}  // namespace loaders
+} // namespace loaders
```

Comment:
Two spaces before comments.  Your clang-format might be old?  Not set correctly?